### PR TITLE
Add readable proposals to governance:view command

### LINKED ISSUE #453: Everyone SBAT easily view the state of Celo governance
### Expected Behavior

A view-only UI exists to see the state of governance, e.g. what is the contents of the queue, how many upvotes have each received, etc.

### Current Behavior

No way of viewing this easily

### PR DESCRIPTION
**Pull request by yorhodes**
### Description

Adds a `raw` flag to `governance:view` which dictates whether proposal should be translated to JSON when outputted

### Tested

Against `yarn test:livechain`
image (2)

### Other changes

Adds optional parameters named map to `ProposalTransactionJSON` interface

### Related issues

- Fixes #453

### CODE REVIEW

File: packages/cli/src/commands/governance/view.ts [hunk 1]
```diff
@@ -9,20 +10,26 @@ export default class View extends BaseCommand {
   static flags = {
     ...BaseCommand.flags,
     proposalID: flags.string({ required: true, description: 'UUID of proposal to view' }),
+    raw: flags.boolean({ required: false, description: 'Display proposal in raw bytes format' }),
```
Comment (nambrot) [inline packages/cli/src/commands/governance/view.ts:13]: What's the default?

File: packages/contractkit/src/wrappers/Governance.ts [hunk 2]
```diff
@@ -342,11 +342,12 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
...
   /**
-   * Returns the proposal dequeue as list of proposal IDs.
+   * Returns the (existing) proposal dequeue as list of proposal IDs.
    */
-  getDequeue = proxyCall(this.contract.methods.getDequeue, undefined, (arrayObject) =>
-    arrayObject.map(valueToBigNumber)
-  )
+  async getDequeue() {
+    const dequeue = await this.contract.methods.getDequeue().call()
+    return dequeue.map(valueToBigNumber).filter((id) => !id.isZero())
```
Comment (nambrot) [inline packages/contractkit/src/wrappers/Governance.ts:350]: When would the IDs be zero?

File: packages/contractkit/src/wrappers/Governance.ts [hunk 3]
```diff
@@ -430,9 +431,11 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
...
   private async lesserAndGreaterAfterUpvote(upvoter: Address, proposalID: BigNumber.Value) {
     const upvoteRecord = await this.getUpvoteRecord(upvoter)
-    const queue = upvoteRecord.proposalID.isZero()
-      ? await this.getQueue()
-      : (await this.withUpvoteRevoked(upvoter)).queue
+    const recordQueued = await this.isQueued(upvoteRecord.proposalID)
+    const queue = recordQueued
+      ? (await this.withUpvoteRevoked(upvoter)).queue
+      : await this.getQueue()
+    console.log(queue)
```
Comment (nambrot) [inline packages/contractkit/src/wrappers/Governance.ts]: Should this be in here?

File: packages/cli/src/commands/governance/view.ts (hunk 1 above)
Comment (yorhodes) [reply packages/cli/src/commands/governance/view.ts:13]: defaults to false (equivalent to excluding as flag in oclif)

File: packages/contractkit/src/wrappers/Governance.ts (hunk 2 above)
Comment (yorhodes) [reply packages/contractkit/src/wrappers/Governance.ts:350]: dequeued indices are reused after proposals are deleted (when expired)
the proposalID is set to zero in `deleteDequeuedProposal`, I will add a comment

File: packages/contractkit/src/wrappers/Governance.ts (hunk 3 above)
Comment (yorhodes) [reply packages/contractkit/src/wrappers/Governance.ts]: nope

**Review by asaj — APPROVED** [review]

**Pull request merged**